docs/sources/datasources/azure-monitor/query-editor/index.md

```diff
@@ -43,20 +46,20 @@ In contrast, Azure Monitor Logs can store a variety of data types, each with the
 **To create a Metrics query:**
 
 1. In a Grafana panel, select the **Azure Monitor** data source.
-1. Select the **Metrics** service.
-1. Select a resource from which to query metrics by using the subscription, resource group, resource type, and resource fields. Multiple resources can also be selected as long as they belong to the same subscription, region and resource type. Note that only a limited amount of resource types support this feature.
-1. To select a different namespace than the default—for instance, to select resources like storage accounts that are organized under multiple namespaces—use the **Namespace** option.
+2. Select the **Metrics** service.
```

Comment:
While Markdown will render a numbered list, we usually use 1. instead of 1., 2., etc. Markdown automatically numbers the steps. For more information, see the [Writers Toolkit](https://grafana.com/docs/writers-toolkit/writing-guide/markdown-guide/#numbered-lists). I have to be careful in VSCode because it will automatically change the numbered lists from 1. to 1., 2.